### PR TITLE
Ccg

### PR DESCRIPTION
**Pull request by NMonteix**

### CODE REVIEW

**Comment by rmzelle** [comment]
Did you remove all those non-breaking spaces (`&#160;`) on purpose?

**Comment by NMonteix** [comment]
No...
Didn't check...
Changes to be done...

**Comment by adam3smith** [comment]
(looks good otherwise)

**Comment by NMonteix** [comment]
It is strange though: I have used the desk version of Github to pull the request; on my local copy of the repository, cahiers-du-centre-glotz.csl do present the non-breaking spaces.
Does any of you have an explanation?
Will send the changes by tomorrow

**Comment by rmzelle** [comment]
@adam3smith, I'm not sure if http://formatter.citationstyles.org/ properly escapes non-breaking spaces. It doesn't seem to do anything if I copy/paste http://www.fileformat.info/info/unicode/char/00a0/browsertest.htm into a style, although I think it should work: `outputString = outputString.replace(/\u00a0/g,"&#160;"); //no-break space`

**Comment by rmzelle** [comment]
@NMonteix, thanks! Did you figure out what went wrong?

File: cahiers-du-centre-gustave-glotz.csl [hunk 1]
```diff
@@ -195,8 +195,19 @@
...
                 <text variable="volume"/>
                 <text variable="issue"/>
-                <text macro="pages"/>
               </group>
+                <choose>
+                  <if variable="page" match="any">
+                    <text macro="pages"/>
+                  </if>
+                  <else>
+                    <group prefix=" [" suffix="]">
```
Comment (adam3smith) [inline cahiers-du-centre-gustave-glotz.csl]: you'll want to test for `<if variable="URL">` here to be safe. It's rare but possible that we have items with access date but no URL.

File: cahiers-du-centre-gustave-glotz.csl [hunk 2]
```diff
@@ -208,37 +219,29 @@
                 <text macro="collection" prefix=" "/>
               </group>
             </else-if>
-            <else-if type="entry-dictionary entry-encyclopedia" match="any">
-              <group delimiter=", " prefix=", ">
-                <text macro="title" quotes="true"/>
+            <else-if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
+              <text macro="title" quotes="true" suffix=", "/>
+              <text term="in" suffix="&#160;"/>
+              <group delimiter=" ">
```
Comment (adam3smith) [inline cahiers-du-centre-gustave-glotz.csl]: why not just put the comma in the delimiter and delete all suffixes below?

**Comment by NMonteix** [comment]
No idea
To correct, I used github online
Could it be linked with XML copy editor?

**Comment by adam3smith** [comment]
that was a bit fast, sorry, easier to see without all the space diffs. @NMonteix -- see my comments inline, might want to address those at some point, so neither is urgent.

**Comment by rmzelle** [comment]
> To correct, I used github online
Could it be linked with XML copy editor?

Yeah, it might unescape the characters. Which isn't really wrong, but it's not what we want for the repository.

**Comment by NMonteix** [comment]
Thanks to you!
I have locally done the changes for the commas, will test for URL tomorrow...

**Pull request merged**